src/main/java/net/trapezokomos/dashboard/exception/ResourceAlreadyExistsException.java

```diff
@@ -0,0 +1,13 @@
+package net.trapezokomos.dashboard.exception;
+
+import java.io.Serial;
+
+public class ResourceAlreadyExistsException extends Exception {
```

Comment:
Lets create an intermediate Exception Class 

GenericAPIException
GenericAPIRunTimeException
and extend through those